### PR TITLE
TBBR cleanups

### PR DESCRIPTION
**Pull request by masahir0y**

### CODE REVIEW

**Comment by davidcunado-arm** [comment]
jenkins: test this please

File: drivers/auth/auth_mod.c [hunk 1]
```diff
@@ -59,9 +58,9 @@ extern unsigned int auth_img_flags[];
 static int cmp_auth_param_type_desc(const auth_param_type_desc_t *a,
 		const auth_param_type_desc_t *b)
 {
-	if ((a->type == b->type) && (a->cookie == b->cookie)) {
+	if (a->type == b->type && a->cookie == b->cookie)
```
Comment (davidcunado-arm) [inline drivers/auth/auth_mod.c]: Although the parenthesis are not needed in this case, having them there makes the code easier to read and more critically there are static analysis rules that TF is looking to support that state that code should not rely on operator precedence (and so use parenthesis to enforce).

Comment (masahir0y) [inline drivers/auth/auth_mod.c]: I do not like your idea, but I am following you
just because I do not want this pull request blocked.

**Comment by masahir0y** [comment]
Updated and pushed.

File: drivers/auth/auth_mod.c [hunk 2]
```diff
@@ -267,38 +262,32 @@ static int auth_nvctr(const auth_method_param_nv_ctr_t *param,
...
-		/* Invalid ASN.1 integer */
+	if (*p != ASN1_INTEGER)		/* Invalid ASN.1 integer */
 		return 1;
-	}
 	p++;
 
 	/* NV-counters are unsigned integers up to 32-bit */
 	len = (unsigned int)(*p & 0x7f);
-	if ((*p & 0x80) || (len > 4)) {
+	if (*p & 0x80 || len > 4)
```
Comment (davidcunado-arm) [inline drivers/auth/auth_mod.c]: @masahir0y Ahhh! Really sorry missed this one in my previous review; apologies! Could you revert the parentheses on this one too?
Thanks!

**Comment by masahir0y** [comment]
Updated again.

**Comment by davidcunado-arm** [comment]
jenkins: test this please

**Comment by danh-arm** [comment]
This PR is "borderline gratuitous". It adds little value but significant code churn. We'll merge this (once David's last comment is addressed) but please avoid changes like this in future, unless they accompany more significant changes. Thanks.

**Comment by masahir0y** [comment]
>This PR is "borderline gratuitous". It adds little value but significant code churn. We'll merge this (once David's last comment is addressed) but please avoid changes like this in future, unless they accompany more significant changes. Thanks.

OK, fine.   Let's not merge this.

**Pull request closed without merging**